packages/server/api/src/app/ee/platform/platform-plan/platform-plan.controller.ts

```diff
@@ -138,6 +138,43 @@ export const platformPlanController: FastifyPluginAsyncTypebox = async (fastify)
         })
     })
 
+    fastify.post('/create-subscription', CreateSubscriptionRequest, async (request) => {
+        const { stripeCustomerId: customerId, ...platformPlan } = await platformPlanService(request.log).getOrCreateForPlatform(request.principal.platform.id)
+        assertNotNullOrUndefined(customerId, 'Stripe customer id is not set')
+
+        const { newActiveFlowsLimit } = request.body
+
+        const baseActiveFlowsLimit = STANDARD_CLOUD_PLAN.activeFlowsLimit ?? 0
+        const extraActiveFlows = Math.max(0, newActiveFlowsLimit - baseActiveFlowsLimit)
+
+        return stripeHelper(request.log).startSubscription({
+            platformId: platformPlan.platformId,
+            customerId,
+            extraActiveFlows,
+        })
+    })
+
+    fastify.post('/update-active-flows-limit', UpdateActiveFlowsLimitRequest, async (request) => {
```

Comment:
Nit: this should be just update for platform_plan object and activeFlows is just in the body.